xchange-coingi/src/main/java/org/knowm/xchange/coingi/service/CoingiMarketDataService.java

```diff
@@ -0,0 +1,63 @@
+package org.knowm.xchange.coingi.service;
+
+import org.knowm.xchange.Exchange;
+import org.knowm.xchange.coingi.CoingiAdapters;
+import org.knowm.xchange.coingi.dto.marketdata.CoingiOrderBook;
+import org.knowm.xchange.currency.CurrencyPair;
+import org.knowm.xchange.dto.marketdata.OrderBook;
+import org.knowm.xchange.dto.marketdata.Ticker;
+import org.knowm.xchange.dto.marketdata.Trades;
+import org.knowm.xchange.service.marketdata.MarketDataService;
+
+import java.io.IOException;
+
+public class CoingiMarketDataService extends CoingiMarketDataServiceRaw
```

Comment:
this class need exception adaptation still

Reply:
done, thanks